### PR TITLE
Update French Channels

### PR DESCRIPTION
**Pull request by sebitr**

### CODE REVIEW

**Comment by sebitr** [comment]
Hi,
Row 3095 looks ok to me. Sorry I don't understand the error.

File: data/channels.csv [hunk 1]
```diff
@@ -3060,28 +3060,31 @@ CanalPlusDomo.pl,Canal + Domo,,,PL,,,c/PL,pol,,FALSE,,https://ocdn.eu/ptv2-image
...
 CanalPlusOuest.fr,Canal + Ouest,,,FR,,,c/FR,fra,,FALSE,,
 CanalPlusPolynesie.fr,Canal + Polynésie,,,FR,,,c/FR,fra,,FALSE,https://www.canalplus.com/,https://www.lyngsat.com/logo/tv/cc/canalplus_fr_polynesie.png
 CanalPlusPopCentre.fr,Canal + Pop Centre,,,FR,,,c/FR,fra,,FALSE,http://www.canalplus-afrique.com/,https://service.canal-overseas.com/image-api/v1/image/82f77219a07c9eb0f43dd77ddf7ed879
 CanalPlusPopOuest.fr,Canal + Pop Ouest,,,FR,,,c/FR,fra,,FALSE,http://www.canalplus-afrique.com/,https://service.canal-overseas.com/image-api/v1/image/f920626326fadd2c35a756b4c836cd6f
 CanalPlusPremiereCentre.fr,Canal + Première Centre,,,FR,,,c/FR,fra,,FALSE,http://www.canalplus-afrique.com/,https://service.canal-overseas.com/image-api/v1/image/3e8e241717f3575e4124e59dceb74f8a
 CanalPlusPremiereOuest.fr,Canal + Première Ouest,,,FR,,,c/FR,fra,,FALSE,http://www.canalplus-afrique.com/,https://service.canal-overseas.com/image-api/v1/image/da40df593cb3e37f73b905d9a0239ed3
+CanalPlusPremierLeague.fr,Canal + Premier League,,,FR,,,c/FR,fra,,FALSE,https://static.wikia.nocookie.net/logopedia/images/2/28/Canal_PremierLeague.svg
```
Comment (freearhey) [inline data/channels.csv:3087]:
```suggestion
CanalPlusPremierLeague.fr,Canal + Premier League,,,FR,,,c/FR,fra,,FALSE,,https://static.wikia.nocookie.net/logopedia/images/2/28/Canal_PremierLeague.svg
```

File: data/channels.csv [hunk 2]
```diff
@@ -3101,6 +3104,7 @@ CanalPlusSportPolska.pl,Canal + Sport Polska,,,PL,,,c/PL,pol,,FALSE,https://tv.p
 CanalPlusSportReunion.fr,Canal + Sport Réunion,,,FR,,,c/FR,fra,,FALSE,http://www.canalplus.fr/pid1747-c-sport.html,https://service.canal-overseas.com/image-api/v1/image/f280baf8d475a4643d64321caf771782
 CanalPlusSportsMyanmar.fr,Canal + Sports Myanmar,,,FR,,,c/FR,fra,,FALSE,,https://www.lyngsat.com/logo/tv/cc/canal-plus-sports-myanmar-fr-mm.png
 CanalPlusSuSann.fr,Canal + Su Sann,,,FR,,,c/FR,fra,,FALSE,,https://www.lyngsat.com/logo/tv/cc/canal-plus-susann-myanmar-fr-mm.png
+CanalPlusTop14.fr,Canal + Top 14,,,FR,,,c/FR,fra,,FALSE,https://static.wikia.nocookie.net/logopedia/images/6/63/Canal_Top14Rugby.svg/
```
Comment (freearhey) [inline data/channels.csv:3107]:
```suggestion
CanalPlusTop14.fr,Canal + Top 14,,,FR,,,c/FR,fra,,FALSE,,https://static.wikia.nocookie.net/logopedia/images/6/63/Canal_Top14Rugby.svg/
```

**Review by freearhey — CHANGES_REQUESTED** [review]
Please check the marked rows again.

**Pull request closed without merging**